contracts/FlowIDTableStaking.cdc

```diff
@@ -728,6 +729,15 @@ pub contract FlowIDTableStaking {
             return <-node
         }
 
+        /// Sets a list of approved node IDs for the next epoch
+        /// Nodes not on this list will be unstaked at the end of the staking auction
+        /// and not considered to be a proposed/staked node
+        pub fun setApprovedList(_ nodeIDs: [String]) {
+            let list = FlowIDTableStaking.account.load<[String]>(from: /storage/idTableApproveList)
+
+            FlowIDTableStaking.account.save<[String]>(nodeIDs, to: /storage/idTableApproveList)
```

Comment:
Is there a reason we need to load it first?

Reply:
If you try to save a value to storage when there is already something there, the save will fail